api/envoy/extensions/filters/listener/tls_inspector/v3/tls_inspector.proto

```diff
@@ -32,4 +32,10 @@ message TlsInspector {
   // tls inspector will consume.
   google.protobuf.UInt32Value initial_read_buffer_size = 2
       [(validate.rules).uint32 = {lt: 65537 gt: 255}];
+
+  // Populate ``JA4`` fingerprint hash using data from the TLS Client Hello packet.
+  // ``JA4`` is an improved version of ``JA3`` that includes TLS version, ciphers, extensions,
+  // and ALPN information in a hex format. Default is false.
+  // [#not-implemented-hide:]
+  google.protobuf.BoolValue enable_ja4_fingerprinting = 3;
```

Comment:
why add this proto option unless we have accompanying code that uses it?

Reply:
I was under the impression that proto changes need to go in first. I updated the PR with the implementation.